tools/tests/test_airport_darwin_10_9.plist

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would keep `test_airport` and express your implicit shame for 10.9 users by calling this `test_airport_legacy.plist` ;)